course_catalog/tasks.py

```diff
@@ -172,7 +172,7 @@ def get_ocw_data(upload_to_s3=True):  # pylint:disable=too-many-locals,too-many-
                 if settings.OCW_UPLOAD_IMAGE_ONLY:
                     parser.upload_course_image()
                 else:
-                    parser.upload_all_media_to_s3()
+                    parser.upload_all_media_to_s3(upload_master_json=True)
```

Comment:
I think this will only be run for courses that have changed since the last check or are have not yet been imported.  I could manually delete all the OCW courses, but it might be useful to add a `force_overwrite` parameter to this function, that if True would run through all courses even if they haven't changed.  But either way, all the media would have to be uploaded again.  Another alternative is to write a django management command to upload the `raw_json` field of each existing OCW course to S3 as the master_json file, assuming it is the same content.  @pdpinch what do you think?

Reply:
I do need all the files, so I'll need someway to recreate them. A management command seems most convenient to me, but I'm flexible.

Reply:
I think a management command makes sense in this case. Should this command check if the file already exists or just naively redo everything every time (though the command may only ever be needed once)?

Reply:
I'm fine with naively redoing all of them, since it will probably just be run once when none exist.

Reply:
Should I do that as part of this PR or as a separate one?

Reply:
You can add it in this PR

Reply:
I have added the management command.